editor/code/package.json

```diff
@@ -111,7 +111,7 @@
         "title": "Coq LSP: try to jump to next Coq sentence"
       },
       {
-        "command": "coq-lsp.sentenceBack",
+        "command": "coq-lsp.sentencePrevious",
         "title": "Coq LSP: try to jump to previous Coq sentence"
```

Comment:
Also whilst we are here, let's change these to `Try` same here and above.

Reply:
Or alternatively drop the try and just start with `Jump`.

Reply:
The thing is that the command really "tries" to jump, it can fail often.

Reply:
Let's continue discussion on #699 , I'll be merge it before next release.